framework/src/org/checkerframework/framework/type/AnnotatedTypeFactory.java

```diff
@@ -883,6 +893,8 @@ public AnnotationFormatter getAnnotationFormatter() {
     /** Mapping from a Tree to its TreePath * */
     private final TreePathCacher treePathCache = new TreePathCacher();
 
+    private boolean withCombineConstraints;
```

Comment:
Yes, made it final, added documentation and dependancy. Changed the name to withViewpointAdaptation